src/psyclone/dynamo0p3.py

```diff
@@ -1878,7 +1878,7 @@ def kern_args(self, stub=False, var_accesses=None):
                 if var_accesses is not None:
                     var_accesses.add_access(Signature(adj_face),
                                             AccessType.READ, self._kernel,
-                                            [[1]])
+                                            [1])
```

Comment:
Yes, but #1320  (while this would be simple, I need to think about this a bit, it might be better to be ":" or so??). I don't think we have any kind of information about how this field is going to be used in the kernel in the first place, at least ":" would indicate that anything can be used.

Reply:
Oh, that's a point - I'd forgotten this was the LFRic API and therefore we won't (yet) necessarily have PSyIR to refer to. However, since it is the LFRic API then we may be able to deduce stuff from the rules.

Reply:
I assume you don't want me to do anything here? I do deduce stuff from the fact that it is LFRic: it is a read-only, 1-d array. I have to admit I am not sure how to get more, I can't know how this argument is used in the kernel, but the important point is that it is not dependent on the psy loop over all columns I think.

Reply:
Please add a comment to say that the `[1]` argument is to flag that the adjacent faces argument is a rank-1 array.

Reply:
Done, and added a todo #1320 as well (likely we might want to use ":" here ... though ... it's not actually clear to me ... do we have any information how the variables that PSyclone adds automatically are used???)

Reply:
Yes, I think we have quite a lot in the User Guide. If there's stuff missing then we should ask the scientists and flesh it out.